test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.default.test.ts

```diff
@@ -82,11 +82,14 @@ describe('parse bpmn as json for default sequence flow', () => {
   });
 
   it(`should NOT convert, when an sequence flow (defined as default) is an attribute of 'process' and attached to a flow node where is NOT possible in BPMN Semantic`, () => {
-    const json = {
+    const json: BpmnJsonModel = {
       definitions: {
         targetNamespace: '',
         process: {
           id: 'Process_1',
+          // To enforce the type and test a case who never should happen
```

Comment:
**praise:** see my previous comment, this cannot happen if the BpmnJsonModel was always created via some code. However, the actual data are generated from the BPMN source and so, such this use case can happen.